packages/codegen-ui/lib/utils/form-component-metadata.ts

```diff
@@ -94,6 +94,7 @@ export const mapFormMetadata = (form: StudioForm, formDefinition: FormDefinition
 
         if (
           (element.relationship ||
```

Comment:
If typescript allows, can we just pare down this condition to 'valueMappings' in element?

Reply:
Yup. itll work